internal/store/files/README.md

```diff
@@ -21,3 +21,24 @@ WHERE
   predicate = 'fips52AlphaCode'
   AND subject_id LIKE 'geoId/%'
 ```
+
+## recon_name_to_types.json generation
+
+1. Use BQ to get the list of names/alternate names and their types for a list of entity types that we do name recon for. Query used for current file:
+
+```sql
+SELECT
+  t1.object_value as name,
+  t2.object_id
+FROM `datcom-store.dc_kg_latest.Triple` as t1
+JOIN `datcom-store.dc_kg_latest.Triple` as t2
+ON t1.subject_id = t2.subject_id 
+WHERE 
+  t1.predicate = 'name' and
+  t2.predicate = 'typeOf' and
+  t2.object_id in ("VirusGenusEnum", "VirusIsolate", "Virus", "Species", "BiologicalSpecimen", "GeneticVariant", "Gene", "Disease", "ICD10Section", "ICD10Code", "MeSHDescriptor", "Drug", "AnatomicalTherapeuticChemicalCode", "MeSHSupplementaryRecord")
```

Comment:
-- This list should match the list [here](http://canonical-path-to-the-list-in-google3?)

Reply:
done